style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
I don't think it can be extrapolated from that, however, I think this wording could be improved. What about:

> When defining methods whose signature exceeds 80 characters, list arguments on separate lines.